build.sh

```diff
@@ -6,12 +6,14 @@ RPM_RELEASE="0.1.$MILESTONE.$(date -u +%Y%m%d%H%M%S)"
 
 ROLE_NAME="oVirt.disaster-recovery"
 PACKAGE_NAME="ovirt-ansible-disaster-recovery"
-PREFIX=/usr/local
+LOG_NAME="ovirt-dr"
+PPREFIX=/usr/local
```

Comment:
Why it's renamed to PPREFIX? I think it's typo, no?

Reply:
ohh damn, you are right, will fix